### PR TITLE
[WIP] Update ember-cli

### PR DESCRIPTION
**Pull request by GavinJoyce**
This pins ember-cli to include https://github.com/ember-cli/ember-cli/pull/7731 so that we can implement more generator blueprints in https://github.com/ember-cli/ember-cli/issues/7530

TODO:

 * [ ] Fix remaining 16 failing tests

### CODE REVIEW

**Comment by GavinJoyce** [comment]
`destroyApp` has been removed: https://github.com/ember-cli/ember-cli/commit/ec12c757fab32d4207f35e4ddd048a4cfba2289a

File: blueprints/acceptance-test/index.js [hunk 1]
```diff
@@ -17,10 +17,6 @@ module.exports = useTestFrameworkDetector({
       testFolderRoot = pathUtil.getRelativeParentPath(options.entity.name, -1, false);
     }
 
-    let destroyAppExists = fs.existsSync(
```
Comment (rwjblue) [inline blueprints/acceptance-test/index.js]: Hmm, I think this still needs to exist no? This isn't about the ember-cli version that is being used by ember-source, but rather about the current project layout _when the blueprint is ran_...

Comment (GavinJoyce) [reply blueprints/acceptance-test/index.js]: You're right, thanks for pointing this out. I enjoy deleting code so much, I just jumped right in 🙈

**Comment by GavinJoyce** [comment]
I'm going to close this and revisit in the morning.

**Pull request closed without merging**